vectormath/vector.py

```diff
@@ -189,18 +189,24 @@ class Vector2(BaseVector):
         - no input (returns [0., 0.])
     """
 
-    def __new__(cls, x=None, y=None):                                          #pylint: disable=arguments-differ
+    def __new__(cls, x=None, y=None, polar=False, unit='rad'):                 #pylint: disable=arguments-differ
 
         def read_array(X, Y):
             """Build Vector2 from another Vector2, [x, y], or x/y"""
             if isinstance(X, cls) and Y is None:
-                return cls(X.x, X.y)
+                return cls(X.x, X.y, polar, unit)
```

Comment:
I don't think we want to pass through `polar` in this case, where we are copying the input vector. If we did, it would transform the vector in an unexpected way. I would probably just ignore the polar/unit args, i.e. `return cls(X.x, X.y)`. Alternatively, you could raise an exception if `polar=True`.